src/qt/intro.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include <qt/guiutil.h>
 
+#include <chainparams.h>
```

Comment:
You can't include core files directly (this is part of ongoing work to separate the two more cleanly); you'll need to go through the node interface (see the include below).

Reply:
I see! Thank you, I will fix this as soon as I get back.